src/hub.html

```diff
@@ -656,6 +655,7 @@
                 class="left-controller"
                 hand-controls2="left"
                 tracked-controls
+                matrix-auto-update
```

Comment:
:( Hopefully we can get rid of this when we get around to ditching tracked-controls and having a component that just reads pose from an action in the input system